src/ray/raylet/local_object_manager.cc

```diff
@@ -231,15 +231,23 @@ void LocalObjectManager::SpillObjectsInternal(
                 num_active_workers_ -= 1;
               }
               io_worker_pool_.PushSpillWorker(io_worker);
-              if (!status.ok()) {
-                for (const auto &object_id : objects_to_spill) {
+              size_t num_objects_spilled = r.spilled_objects_url_size();
```

Comment:
Can you add a Cpp unit test to verify this?

Reply:
@rkooo567  could you give me more details about verifying what? Did you mean if partial succeed, the rest will be re-tried later?

Reply:
Yes! That's right!